shared/src/clients/phonebook-client.class/index.ts

```diff
@@ -0,0 +1,62 @@
+import axios, { AxiosInstance, AxiosRequestConfig } from 'axios'
+
+export default class PhonebookClient {
+  private client: AxiosInstance
+  private baseUrl: string
+  private apiKey: string
+  private version: string
+
+  constructor(baseUrl: string, apiKey: string, version: string) {
+    this.baseUrl = baseUrl
+    this.apiKey = apiKey
+    this.version = version
+    this.client = axios.create({
```

Comment:
[suggestion] Can also add `keepAlive` functionalities to reuse the same connection instead of establishing new one to phonebook everytime https://stackoverflow.com/questions/69889711/does-axios-support-keepalive-functionality-same-as-the-native-fetch